src/less/mixins/private/bubble-mixins.less

```diff
@@ -1,10 +1,13 @@
 // this mixin is used by infotip, tooltip, tourtip
 
 @bubble-base-box-shadow: 0 0 3px rgba(17, 24, 32, 0.499);
```

Comment:
Is the base-box-shadow not used?

Reply:
It's used in this file.

Reply:
Yeah it's used like this:
` box-shadow: var(--bubble-base-box-shadow, @bubble-base-box-shadow);` 
We should probably add that as a variable in `:root` and not use the second one. Otherwise the first is not being used

Reply:
Done